src/app/donation-complete/donation-complete.component.html

```diff
@@ -25,17 +25,19 @@ <h2>Thank you!</h2>
 
       <div id="save-my-details-cta" *ngIf="!registrationComplete" >
         <div class="button-wrapper">
+          <!-- Spaces for expanding button are a bit of an ugly hack, but maybe worthwhile as lonng as we have no other buttons that need expanding-->
           <biggive-button
             space-below="0"
             colour-scheme="white"
             is-past-campaign="false"
             is-future-campaign="false"
-            label="Save my details"
+            label="&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Save my details&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;"
```

Comment:
@bdsl Isn't this what the `full-width` attribute it meant for?

Reply:
I'm not 100% sure, but in my test adding `fullWidth="true"` here doesn't seem to do anything to make the button wide if it doesn't have the spaces.

Reply:
@bdsl When using as a component attribute in HTML, you need the dash-case version. https://stenciljs.com/docs/properties#variable-casing

I'll make a PR which hopefully lets us do this without the hack, assuming the goal is to make the button 100% width following the existing component library style.

Reply:
https://github.com/thebiggive/donate-frontend/pull/890